packages/studio-plugin/src/utils/TypeGuards.ts

```diff
@@ -165,54 +161,31 @@ export default class TypeGuards {
     return typeof value === "string" && value.startsWith("siteSettings.");
   }
 
-  static isNotFragmentElement(
-    element: JsxElement | JsxSelfClosingElement | JsxFragment
-  ): element is JsxElement | JsxSelfClosingElement {
-    if (element.isKind(SyntaxKind.JsxFragment)) {
-      return false;
-    }
-    if (element.isKind(SyntaxKind.JsxSelfClosingElement)) {
+  static isExpressionElement(
+    element: JsxElement | JsxSelfClosingElement | JsxFragment | JsxExpression
+  ): element is JsxExpression {
+    if (element.isKind(SyntaxKind.JsxExpression)) {
       return true;
     }
-    const name = StaticParsingHelpers.parseJsxElementName(element);
-    return !["Fragment", "React.Fragment"].includes(name);
-  }
-
-  static isModuleMetadata(
-    metadata?: FileMetadata | null
-  ): metadata is ModuleMetadata {
-    return metadata?.kind === FileMetadataKind.Module;
-  }
-
-  static isModuleState(
-    componentState: ComponentState
-  ): componentState is ModuleState {
-    return componentState.kind === ComponentStateKind.Module;
-  }
-
-  static isStandardOrModuleComponentState(
-    componentState: ComponentState
-  ): componentState is StandardOrModuleComponentState {
-    return (
-      componentState.kind === ComponentStateKind.Module ||
-      componentState.kind === ComponentStateKind.Standard
-    );
+    return false;
   }
 
-  static isRepeaterState(
-    componentState: ComponentState
-  ): componentState is RepeaterState {
-    return componentState.kind === ComponentStateKind.Repeater;
+  static isFragmentElement(
+    element: JsxElement | JsxSelfClosingElement | JsxFragment | JsxExpression
+  ): element is JsxFragment {
+    if (element.isKind(SyntaxKind.JsxFragment)) {
+      return true;
+    } else if (element.isKind(SyntaxKind.JsxElement)) {
+      const name = StaticParsingHelpers.parseJsxElementName(element);
+      return ["Fragment", "React.Fragment"].includes(name);
+    }
+    return false;
   }
 
-  static isEditableComponentState(
+  static isStandardComponentState(
```

Comment:
This type guard doesn't seem very helpful anymore since it just checks for one type now